docs/articles/documentation/templates/actions/selector-parameter.md

```diff
@@ -121,23 +81,13 @@ You can pass any of the following objects as a `selector`.
 
 Before executing an action, TestCafe waits for the target element to appear
 in the DOM and become visible. If this does not happen
-within the [selector timeout](../selecting-page-elements/selectors/using-selectors.md#selector-timeout), the test fails.
+within the [selector timeout](../../guides/basic-guides/select-page-elements.md#selector-timeout), the test fails.
 
 Note that TestCafe cannot interact with page elements overlaid by a different element.
 If the target element is not on top when an action is triggered, TestCafe waits for this element to appear in the foreground.
-If this does not happen within the [selector timeout](../selecting-page-elements/selectors/using-selectors.md#selector-timeout),
+If this does not happen within the [selector timeout](select-page-elements.md#selector-timeout),
 the action is performed with an overlaying element. To learn why the target element can be overlaid,
 see the *stacking* description in the [z-index](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index) topic.
 
-> An exception is the [Upload action](upload.md). It does not require the target `input` to be visible.
+> An exception is the [Upload action](../../reference/test-api/testcontroller/upload.md). It does not require the target `input` to be visible.
```

Comment:
"An exception is the [Upload action]"? Explain

Reply:
@dirk-pieterse We should probably rephrase this. This is, like most articles in this pull request, a part of old documentation that definitely needs not only restructuring but also new descriptions altogether. We should highlight that `upload action` is the only method that does not require its target element to be visible.